apps/builder/app/builder/features/style-panel/shared/model.tsx

```diff
@@ -297,14 +294,16 @@ export const $definedComputedStyles = computed(
         state: styleSourceSelector?.state,
         property,
       });
+      computedStyleDecl.listed = listed;
```

Comment:
a bit unexpected place for it

Reply:
ok I tried, but couldn't find a better solution for now